src/components/IconToggle/README.md

```diff
@@ -0,0 +1,48 @@
+## IconToggle
+An implementation of an IconToggle for TPAs,
+The TPA implementation provides few extendable theme properties with the ability to override the base properties as well,
+
+By default the IconToggle will pull the theme from the site (site colors),
+
+## Theme properties (All are optional)
+
+| propName   | propType | defaultValue | description |
+|------------|----------|--------------|-------------|
+| IconColor  | string   | none | The color of the icon instead of the default color set by theme |
+| LabelColor  | string   | none | The color of the label instead of the default color set by theme |
+| LabelFont  | string   | none | The font of the label instead of the default font |
+
```

Comment:
Note that the default values of the colors and fonts should be as written here:
https://zeroheight.com/7sjjzhgo2/p/05a744/t/7933bc